lib/wildland_dev_tools/heroku.rb

```diff
@@ -50,6 +50,16 @@ def deploy_master_to_staging(verbose = false)
         end
       end
 
+      def import_production_database(verbose = false)
+        puts 'Determining heroku app names.' if verbose
+        production_app_name = get_app_name('production', verbose)
+        scratch_file_name = 'latest.dump'
+        database_name = Rails.configuration.database_configuration['development']['database']
+        download_database(production_app_name, scratch_file_name, verbose)
+        import_database(database_name, scratch_file_name, verbose)
+        File.delete(scratch_file_name)
+      end
```

Comment:
Passing around the state here screams the need for encapsulation into a class and not a module. For example, verbose, app name, database name, etc...